develop/plone/i18n/language.rst

```diff
@@ -248,7 +248,7 @@ Other
 
 * http://blogs.ingeniweb.com/blogs/user/7/tag/i18ndude/
```

Comment:
```suggestion
```

Or this garbage: https://web.archive.org/web/20090115145555/http://blogs.ingeniweb.com/blogs/user/7/tag/i18ndude/